api/src/main/java/io/lumeer/api/dto/JsonQuery.java

```diff
@@ -111,31 +158,43 @@ public boolean equals(final Object o) {
       if (this == o) {
          return true;
       }
-      if (!(o instanceof Query)) {
+      if (!(o instanceof JsonQuery)) {
```

Comment:
`Query` interface was deliberately used here in order to be able to compare the instance with `MorphiaQuery` as well. It may not be needed anymore but I still wonder why this change?

Reply:
This was auto-generated. I will fix that.